easybuild/easyconfigs/n/nsync/nsync-1.29.2-GCCcore-13.2.0.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'CMakeNinja'
+
+name = 'nsync'
+version = '1.29.2'
+
+homepage = 'https://github.com/google/nsync'
+description = """nsync is a C library that exports various synchronization primitives, such as mutexes"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['https://github.com/google/nsync/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['1d63e967973733d2c97e841e3c05fac4d3fa299f01d14c86f2695594c7a4a2ec']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.27.6'),
+    ('Ninja', '1.12.1'),
```

This toolchain has Ninja 1.11.1:
```suggestion
    ('Ninja', '1.11.1'),
```